app/views/whitehall_migration/documents.html.erb

```diff
@@ -1,6 +1,38 @@
-<h1>Documents</h1>
+<% content_for :back_link, render_back_link(href: whitehall_migration_path(@whitehall_migration)) %>
+<% content_for :title, "Document Imports" %>
 
-<% @documents.each do |document| %>
-  <li>ID: <%= document.id %></li>
-  <li>Status: <%= document.state %></li>
-<% end %>
+<% states = ["imported", "sync_failed", "completed"] %>
+
+<% items = @documents.map do |document_import|
+    {
+      link: {
+        text: states.include?(document_import.state) ? document_import.document.current_edition.title : document_import.id.to_s,
+        path: whitehall_migration_document_path(document_import.whitehall_migration_id, document_import),
+      },
+      metadata: {
+        status: document_import.state.to_s,
+        error_message: document_import.error_log ? "Error log: #{document_import.error_log}" : ""
+      },
+      subtext: states.include?(document_import.state) ? "View: <a href=\"#{Plek.find('whitehall-admin')}/government/admin/by-content-id/#{document_import.content_id}\">Whitehall</a> View: <a href=\"#{Plek.find('content-publisher')}/documents/#{document_import.content_id}:en\">Content Publisher</a>".html_safe : "View: <a href=\"#{Plek.find('whitehall-admin')}/government/admin/by-content-id/#{document_import.content_id}\">Whitehall</a>".html_safe,
```

Comment:
I'd suggest setting up these links before hand and then injecting them.

For example:

```
<%
  items = @documents.map do |document_import|
    view_links = []
    if document_import.content_id
      url = "#{Plek.find('whitehall-admin')}/government/admin/by-content-id/#{document_import.content_id}"
      view_links << link_to("Whitehall", url, target: "_blank")
    end
    
    if document_import.document
      view_links << link_to("Content Publisher",
                            document_path(document_import.document), 
                            target: "_blank")
    end
    
    if document_import.completed?
      view_links << link_to("Draft GOV.UK",
                            edition_preview_url(document_import.document.current_edition)
                            target: "_blank")
    end

    if document_import.completed? && document_import.document.live_edition
      view_links << link_to("Live GOV.UK",
                            edition_public_url(document_import.document.live_edition)
                            target: "_blank")
    end
    
    # ...
    
      subtext: view_links.any? ? "View: #{view_links.map(&:html_safe).join(" ")}" : nil,
    }
  end
%>
```

I also snuck in the concept of draft and live gov.uk links above as I realised not everything will be on live GOV.UK